addons/project/views/project_views.xml

```diff
@@ -746,19 +746,19 @@
             <field name="model">project.project</field>
             <field name="arch" type="xml">
                 <form string="Project">
-                    <div class="oe_title">
+                    <div class="oe_title mb-lg-3 mb-md-2">
                         <label for="name" string="Name"/>
                         <h1>
-                            <field name="name" class="o_project_name oe_inline" placeholder="e.g. Office Party"/>
+                            <field name="name" class="o_project_name" placeholder="e.g. Office Party"/>
                         </h1>
                     </div>
                     <field name="user_id" invisible="1"/>
-                    <div class="row mt16 o_settings_container"/>
-                    <div name="alias_def" colspan="2" attrs="{'invisible': [('alias_domain', '=', False)]}">
-                        <label for="alias_name" class="oe_inline mt-4" string="Create tasks by sending an email to"/>
+                    <div class="row o_settings_container" invisible="1"/>
```

Comment:
I am wondering if it is always necessary, can you check? 🤔

If yes, can you explain me why? 🙂 

Thanks in advance! 🙂

Reply:
I made that invisible because it was adding extra space b/w `name` and `alias` [Video](https://nimb.ws/8Qdwhj) when timesheet and planning are not installed
I'll make it visible cause it is not making huge difference or Do I have to confirm with @aju-odoo first
Thank you :slightly_smiling_face:

Reply:
The extra space was not because of `mt16` but now since you remove that class, I don't think the div takes any place. 🤔 

So, I think we could let that div visible. 🤔

Reply:
Oh sorry I did not see your video. 🙈

Reply:
That's strange I don't have the same behavior. 🤔

Reply:
That was happening because of margin overlapping, so putting empty div will actually have better results :slightly_smiling_face:

Reply:
Oh okay, I see, good news. 🙂 

Thanks! 😁